Tests/Tests/RouterSpec.swift

```diff
@@ -411,6 +537,24 @@ class RouterSpec: QuickSpec {
                         scrollView.removeFromSuperview()
                     }
 
+                    it("will insert a tooltip in the same UIWindow type view as the targetView") {
+                        let uiWindow = UIWindowSubclass()
```

Comment:
a normal `UIView` wouldn't be enough for this test?

Reply:
I want to cover the UIWindow case on this switch case
```swift
switch superview {
        case is UIScrollView:
            return superview
        case is UIWindow:
            guard accessibilityCompatibleDisplay,
               let transitionViewClass = NSClassFromString("UITransitionView"),
               let transitionView = superview.subviews.first(where: { !$0.isKind(of: transitionViewClass) }) else {
                return superview
            }
            return transitionView
        default:
            return findParentViewForTooltip(superview)
        }
```

The UIView will not cover that case.